### PR TITLE
Lammps

### PR DESCRIPTION
**Pull request by jaclark5**
## Description
Add data set to test LAMMPS parser

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [X] Add lammps dataset with description
  - [X] Add tests
  - [X] Add documentation

## Status
- [X] Ready to go

### CODE REVIEW

**Comment by jaclark5** [comment]
@xiki-tempula are you the one to request a review from?

File: docs/index.rst [hunk 1]
```diff
@@ -71,7 +71,9 @@ suite it may contain bugs. Please raise any issues or questions in the
 
     gmx
     amber
-    namd
+    gmx
```
Comment (xiki-tempula) [inline docs/index.rst:74]: Duplication?

**Review by xiki-tempula — CHANGES_REQUESTED** [review]
small comments.
Have you read https://alchemtest.readthedocs.io/en/latest/contributing.html and check this PR has done all of the things listed?
Have you download the branch as a archive zip and check that the files are there? (Sometimes test files are not packed properly in the zip)

**Comment by jaclark5** [comment]
Hey @xiki-tempula  I reread the contributing document and noticed that the directions to compile the docs don't align.

I've also compressed the text files and fixed that issue with the docs that you noted.

**Review by xiki-tempula — APPROVED** [review]
LGTM thanks

**Pull request merged**